packages/itwinui-react/src/core/Tile/Tile.tsx

```diff
@@ -120,6 +131,21 @@ export type TileProps = {
   isDisabled?: boolean;
 } & React.ComponentPropsWithoutRef<'div'>;
 
+type TileActionOwnProps = {}; // eslint-disable-line
+
+export const TileAction = (
+  props: PolymorphicComponentProps<'a', TileActionOwnProps>,
+) => {
+  const tileContext = useSafeContext(TileContext);
+
+  React.useEffect(() => {
+    console.log('action useefect');
+    !tileContext.isDisabled ? tileContext.setActionable(true) : null;
```

Comment:
i've realized we can do this with `:has` and avoid an extra rerender (in new chrome/safari) and then use this approach only if `:has` is not supported. see comment here: https://github.com/iTwin/iTwinUI/pull/1127#discussion_r1134494887

also, i think there is no need to check `isDisabled` here. Tile can be both actionable and disabled at the same time, and disabled styling should take preference.

Reply:
Added check for ":has" functionality.